README.md

```diff
@@ -1 +1,60 @@
 # dm-borrower-frontend
+
+The Borrower Frontend provides a website for Borrower to view and sign their
+mortgage deeds online.
+
+## Contents
+- [Usage](#usage)
+- [Getting Started](#getting-started)
+- [Acceptance Tests](#acceptance-tests)
+
+## Usage
+```
+GET     /health                         -- renders standard Gov UK template
+GET     /searchdeed                     -- borrower views deed landing page
+POST    /searchdeed/search              -- form POST the response is a html page which will display the deed or an error page
```

Comment:
Dont they all return HTML pages though? /health and /searchdeed give you html page, its a frontend, you always expect the response to be a page